.gitignore

```diff
@@ -0,0 +1,2 @@
+*.crt
```

Comment:
This seems based on your development environment, not sure if we should commit it..

Reply:
I put it there to make sure no certificate or keys get commited. it's not for my dev env specifically, just a precaution. I can remove it if you think it is unneeded

Reply:
I'd rather not have it here.